openreview/tools.py

```diff
@@ -703,6 +698,37 @@ def post_submission_groups(client, conference_id, submission_invite, chairs):
             readers=[conference_id, chairs],
             signatories=[]))
 
+def get_submission_invitations(client, open_only=False):
+    '''
+    Returns a list of invitations ids visible to the client caccording to the parameter "open_only".
+    
+    :arg client: Object of :class:`~openreview.Client` class
+    :arg open_only: Default value is False. This is a boolean param with value True implying that the results would be invitations having a future due date.
+    
+    Example Usage:
+    >>> get_submission_invitations(c,True)
+    [u'machineintelligence.cc/MIC/2018/Conference/-/Submission', u'machineintelligence.cc/MIC/2018/Abstract/-/Submission', u'ISMIR.net/2018/WoRMS/-/Submission', u'OpenReview.net/Anonymous_Preprint/-/Submission']
+    '''
+    
+    try:
+        #Calculate the epoch for current timestamp
+        now = int(time.time()*1000)
+        
+        if open_only == True:    
+            invitations = client.get_invitations(regex='.*/-/.*[sS]ubmission.*',minduedate=now)
+        else:
+            invitations = client.get_invitations(regex='.*/-/.*[sS]ubmission.*')
+        
+        # For each group in the list, append the invitation id to a list
+        invitation_ids = []
+        
+        for inv in invitations :
```

Comment:
There is a nice way to iterate a list in a single line with Python called [for comprehension](https://docs.python.org/2/tutorial/datastructures.html)

```
invitation_ids = [i.id for i un invitations]
```

Reply:
All 3 changes have been applied.